Hardware/Rpi4.md

```diff
@@ -0,0 +1,98 @@
+---
+arm_hardware: true
+cmake_plat: rpi4
+xcompiler_arg: -DAARCH64=1
+platform: Raspberry Pi 4-b
+arch: ARMv8A
+virtualization: ARM HYP
+iommu: "No"
+soc: BCM2711
+cpu: Cortex-A72
+Status: Unverified
+Contrib: "[Hensoldt](https://hensoldt-cyber.com)"
```

Comment:
We have put some work in the RasPi4 port and unifying the RasPi family support. But some credits should also go to https://gitlab.com/arm-research/security/icecap/icecap then.